runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/QuarkusHttpConnector.java

```diff
@@ -101,8 +105,19 @@ private HttpMethod getMethod(String methodAsString) {
         }
     }
 
+    private HttpVersion getProtocolVersion(String versionAsString) {
+        try {
+            return HttpVersion.valueOf(versionAsString);
+        } catch (IllegalArgumentException e) {
+            String error = "Unsupported HTTP protocol version: " + versionAsString + ". The supported versions are: "
+                    + Arrays.toString(HttpVersion.values());
+            log.warn(error, e);
+            throw new IllegalArgumentException(error);
```

Comment:
After reviewing this again, Im wondering if it wont be better to have this

```suggestion
            log.warn("Unsupported HTTP protocol version: {}. The supported versions are: {}", versionAsString, HttpVersion.values());
            throw e;
```
So basically you want to log that the version is not supported (without the statck trace), but you just throw the original exception in case someone want to deal with it (tipically the stack trace will be printed by an upper layer.